src/Serilog.Sinks.Skywalking/README.md

```diff
@@ -0,0 +1,49 @@
+# serilog-sinks-skywalking
```

Comment:
An English document is required in this repository as SkyWalking and this agent have global users.